src/sgame/sg_main.cpp

```diff
@@ -2140,10 +2140,24 @@ void G_RunFrame( int levelTime )
 		return;
 	}
 
+	int matchTime = levelTime - level.startTime;
+
+	/* Don't process frame at matchTime 0 as delay will be zero and
+	some computation will divide by zero in G_EvaluateAcceleration.
+
+	At map start matchTime == 0 and levelTime == 0.
+	After a map restart matchTime == 0 but levelTime != 0.
+
+	Testing matchTime is enough, we don't need to test levelTime. */
+	if ( !matchTime )
```

Comment:
This seems scary. A lot of really important stuff happens in the first frame; I wouldn't want to skip it.